cumulusci/cumulusci.yml

```
@@ -624,6 +624,7 @@ flows:
                 task: run_tests
             5:
                 task: github_automerge_feature
+                when: project_config.repo_branch.startswith(project_config.project__git__prefix_feature)
```

```suggestion
                when: (project_config.repo_branch or "").startswith(project_config.project__git__prefix_feature)
```

to handle the case where repo_branch is None because the user has checked out a specific commit rather than a branch